### ISSUE TITLE
Compatibility with WPML - Flags do not show when language switcher in menu

### ISSUE THREAD

**Issue by nickihastings** [understrap/understrap#1468]
**Describe the problem**
I am using understrap with WPML and I am seeing a similar issue to this one https://github.com/understrap/understrap/issues/1305 with Polylang. When I activate a language switcher for a menu the image tags for the flags are being stripped out, it is also doing the same thing in the admin where it would usually show a preview.

I notice in the issue for Polylang that your last note implies that everything is working with WPML, so I am just checking whether this is something you are aware of with WPML.

**Screenshots / Videos**
![backend-flags](https://user-images.githubusercontent.com/30271858/137370600-c724b6d7-5a35-4e46-a653-2c2640bab2e7.jpg)
![frontend-flags](https://user-images.githubusercontent.com/30271858/137370612-2692e518-64e5-4587-9f41-b09e6b0cd80e.jpg)

**Comment by bacoords** [comment]
This was added for the next release (v1.1.0) https://github.com/understrap/understrap/commit/7ff3bbdf8c9599f5ee14520125371ba906ab0318

**Comment by bacoords** [comment]
Closed with release [1.1.0](https://github.com/understrap/understrap/releases/tag/v1.1.0).

**Comment by lbriet** [comment]
1.1.0 still breaks flag display for the [Polylang](https://wordpress.org/plugins/polylang/) translation plugin. The image tag _does_ render now, but it filters out the `data:` component in the base64 encoded image src;

i.e.

the following is output

```
<img src="image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAALCAMAAABBPP0LAAAAY1BMVEX/AAD8AADzAADqAAD/fYz+dYT6aHr7XG36VGb6R1f4L0H/ECz+mKXw8fH9/f36+vr19vbW1tbS0tIAG493n89cjcZNgsA/eLkzcbUpaLAcYKwAAFEAA4ANU6UAADsAAHgAAGcSgiW+AAAAS0lEQVR4AQXBiQ3CQBAAMc/dConQf688SoIdBIIyQiR9R0tCuM2rNxHpMqsDIWiBEE4NgnBiHhVJcs48P0uSjXsPl4hLmH2EHzb4A7DPDGppZMSwAAAAAElFTkSuQmCC" alt="Nederlands" width="16" height="11" style="width: 16px;height: 11px">
```

instead of

```
<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAALCAMAAABBPP0LAAAAY1BMVEX/AAD8AADzAADqAAD/fYz+dYT6aHr7XG36VGb6R1f4L0H/ECz+mKXw8fH9/f36+vr19vbW1tbS0tIAG493n89cjcZNgsA/eLkzcbUpaLAcYKwAAFEAA4ANU6UAADsAAHgAAGcSgiW+AAAAS0lEQVR4AQXBiQ3CQBAAMc/dConQf688SoIdBIIyQiR9R0tCuM2rNxHpMqsDIWiBEE4NgnBiHhVJcs48P0uSjXsPl4hLmH2EHzb4A7DPDGppZMSwAAAAAElFTkSuQmCC" alt="Nederlands" width="16" height="11" style="width: 16px;height: 11px">
```

which then causes a 404; 

```
GET https://localhost/nl/home-nl/image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAALCAMAAABBPP0LAAAAmVBMVEViZsViZMJiYrf9gnL8eWrlYkjgYkjZYkj8/PujwPybvPz4+PetraBEgfo+fvo3efkydfkqcvj8Y2T8UlL8Q0P8MzP9k4Hz8/Lu7u4DdPj9/VrKysI9fPoDc/EAZ7z7IiLHYkjp6ekCcOTk5OIASbfY/v21takAJrT5Dg6sYkjc3Nn94t2RkYD+y8KeYkjs/v7l5fz0dF22YkjWvcOLAAAAgElEQVR4AR2KNULFQBgGZ5J13KGGKvc/Cw1uPe62eb9+Jr1EUBFHSgxxjP2Eca6AfUSfVlUfBvm1Ui1bqafctqMndNkXpb01h5TLx4b6TIXgwOCHfjv+/Pz+5vPRw7txGWT2h6yO0/GaYltIp5PT1dEpLNPL/SdWjYjAAZtvRPgHJX4Xio+DSrkAAAAASUVORK5CYII= 404 (Not Found)
```

**Comment by bacoords** [comment]
I'm thinking I'm going to open a PR that just removes the filter on `get_the_title()` completely. I don't see the value of it and I don't even see it as a function in underscores anymore. 

I'll open the PR so others can comment on it, and hopefully we can deprecate `understrap_kses_title()` completely (with a nice fallback of course).

**Comment by kirannasim** [comment]
We can fix this issue to add the "data" protocol to the array of allowed protocols because it is not included by default:

function allow_data_protocol( $protocols ){
    array_push( $protocols, 'data' );
    return $protocols;
}
add_filter( 'kses_allowed_protocols' , 'allow_data_protocol' );

Or we can remove the filter on get_the_title() completely.

**Comment by kirannasim** [comment]
@bacoords Just opened PR https://github.com/understrap/understrap/pull/1648 with updates

**Comment by Sanyastiy** [comment]
> We can fix this issue to add the "data" protocol to the array of allowed protocols because it is not included by default:
> 
> function allow_data_protocol( $protocols ){ array_push( $protocols, 'data' ); return $protocols; } add_filter( 'kses_allowed_protocols' , 'allow_data_protocol' );

@kirannasim Thank you very much! This code just helped me)

**Comment by IanDelMar** [comment]
Fixed in v1.2.0.

@bacoords This and the other issues that where addressed in v1.2.0 can be closed.

**Comment by bacoords** [comment]
🥳 This should be good to go!
